CoffeeChat/Controller/InterestsGroupsViewController.swift

```diff
@@ -0,0 +1,206 @@
+//
+//  InterestsGroupsViewController.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 10/1/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class InterestsGroupsViewController: UIViewController {
```

Comment:
It is quite similar but also slightly different so I'm a bit hesitant to just make 1 view controller. But I do think in general, across all the onboarding screens, things can be standardized and combined... Do you think this can be done in our revision phase or do you think I should try to address in this PR / soon?

Reply:
we can hold off and wait to refactor -- don't worry about it in this PR